tests/unit/EffectiveDateTime.spec.ts

```diff
@@ -245,7 +245,7 @@ describe('Effective Date Time component', () => {
     expect(getLastEvent(wrapper, 'valid')).toEqual(false)
   })
 
-  // this test runs fine locally but fails in GH CI action
+  // FUTURE: this works locally but not in GHA; fix later
```

Comment:
Have we tried this with the new test version :)

Reply:
It's not a test util issue. I believe this is because GH doesn't have the "Pacific" locale that we use to convert dates/times.